docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md

```diff
@@ -82,27 +107,53 @@ We can share the required credentials mentioned in the preceding documentation o
+Please note Spryker is only issuing SSL certificates for endpoints that are managed by Spryker Commerce Cloud.
 
-## User access
+## User access\*
 
 The following is a list of customer or partner users that must have access:
 * AWS Console access: This access can be used to access environment CloudWatch logs, deployment pipelines, parameter store, S3 buckets. Provide the email address of users that need access.
 * VPN: This access can be used to access services such as databases, Jenkins, and RabbitMQ. It is usually needed for developers or DevOps personnel. Provide the email address of users that need access.
-* SSH: This access can be used to access the Bastion Host, and from there, reach other services via port forwarding. This is usually needed in special cases for developers or DevOps personnel. Provide an SSH public key and email address (to use SSH, you require VPN access).
-* SFTP: This access is required to SFTP bastion host. Provide an SSH public key and email address (to SFTP it also requires VPN access) 
-
-
-
-
-
+* SSH: This access can be used to access the Bastion Host, and from there, reach other services via [port forwarding](/docs/cloud/dev/spryker-cloud-commerce-os/access/connecting-to-services-via-ssh.html). This is usually needed in special cases for developers or DevOps personnel. Provide an SSH public key and email address (to use SSH, you require VPN access).
+* SFTP: This access is required to SFTP bastion host. Provide an SSH public key and email address (to SFTP it also requires VPN access). Please note SFTP is not provisioned by default.
+
+## Additional Attributes
+
+In the following we want to explain what additional attributes you can specify at the beginning of your provisioning already, as well as accesses you can request.
+
+### SFTP
+Spryker implemented SFTP on top of EFS.This can be used in case of any third party integrations or for explicit data-imports via Jenkins jobs if required on project level. Note that SFTP is only available on bastion and Jenkins. This feature is disabled by default. This can also be requested later via support ticket, but it is preferred to validate this option during provisioning.
+
+### Site to Site VPN
+A Site-to-Site VPN (Virtual Private Network) is a type of network connection that enables secure communication between two or more geographically separated networks. This type of VPN allows two or more sites to establish a secure and encrypted connection over the internet or other public networks, creating a virtual private network between the two sites.
+- customer gateway IP address
+- IP ranges on the customer side that would need access to Spryker VPC
+Optional:
+- Device name on customer gateway
+- Custom BGP(Border Gateway Protocol) ASN
+- Dynamic BGP routes
+- Other Parameters of tunnels
```

Comment:
@michalwroblewski-spryker what do list items lines 140 through 146 mean?

Reply:
Those are configuration parameters that are required (or optional) to setup Site to Site VPN tunnel between our Spryker Cloud environment and Customer on-premise